### DISCUSSION TITLE
Step 8 script `Build ipa for distribution` exited with status code 65

### DISCUSSION THREAD

**Discussion by haryonorz** [codemagic-ci-cd/codemagic-docs#2179] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

```
Using Xcode 13.4.1 (13F100)
Check D2D.xcworkspace build settings
Execute "xcodebuild -workspace /Users/builder/clone/ios/D2D.xcworkspace -scheme D2D -showBuildSettings"

Archive D2D.xcworkspace
Execute "xcodebuild -workspace /Users/builder/clone/ios/D2D.xcworkspace -scheme D2D -destination generic/platform=iOS -archivePath build/ios/xcarchive/D2D_em9q0enf.xcarchive archive CODE_SIGNING_REQUIRED=YES CODE_SIGNING_ALLOWED=NO DEVELOPMENT_TEAM=RAT64M5AWC 'CODE_SIGN_IDENTITY=Apple Development'"

    Building targets in manual order is deprecated - choose Dependency Order in scheme settings instead, or set DISABLE_MANUAL_TARGET_ORDER_BUILD_WARNING in any of the targets in the current scheme to suppress this warning
Failed to archive /Users/builder/clone/ios/D2D.xcworkspace


Build failed :|
Step 8 script `Build ipa for distribution` exited with status code 65
```

<img width="456" alt="Screenshot 2023-11-08 at 10 45 50" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/45280932/9a167932-ea7b-4005-bbed-bdd7cc4bde20">

Can someone please help? Why is this happening? What can I do to further debug this?

### Build id (optional)

654afdb63a76a9120f455d1a

**Comment by dtrdic** [comment]
hey @haryonorz i can see in the logs that the issue seems to be related to the Yoga library in your Pods. Looks like it's unable to open the Yoga release configuration file.
`/Users/builder/clone/ios/Pods/Target Support Files/Yoga/Yoga.release.xcconfig:1:1: error: unable to open file (in target "Yoga" in project "Pods")`
 Can you check if the Yoga library files are present in your project and if the paths in your project configuration are correct? Maybe updating your Pods could help?

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion due to a lack of activity. You can open the discussion again by replying to this message or opening a new discussion.
